app/population/steps/edit.js

```diff
@@ -0,0 +1,26 @@
+const EditController = require('../controllers/edit')
+
+module.exports = {
+  '/': {
+    entryPoint: true,
+    reset: true,
+    resetJourney: true,
+    skip: true,
+    forwardQuery: true,
+    next: 'details',
+  },
+  '/details': {
+    controller: EditController,
+    pageTitle: 'population::free_space_details.page_title',
+    fields: [
+      'operational_capacity',
+      'usable_capacity',
+      'unlock',
+      'bedwatch',
+      'overnights_in',
+      'overnights_out',
+      'out_of_area_courts',
+      'discharges',
+    ],
```

Comment:
You might want to update the button text for this step. I noticed it currently uses the default of "Continue" which suggests there's another step.

We currently have a `confirm_and_save` action but might want to check with @Andy-Millar on what's the best copy here. Maybe just "Save".

But once you have the action key you can add this line:

```
'/details': {
  ...
  buttonText: 'actions::confirm_and_save',
  ...
}
```

Reply:
Good idea - set it to use "Confirm and Save" for now, if @Andy-Millar wants it changed it's easy enough to tweak later :)